test/test.mjs

```diff
@@ -907,13 +907,10 @@ async function startBrowser({
     options.args = ["--no-sandbox", "--disable-setuid-sandbox"];
     // silent printing in a pdf
     options.args.push("--kiosk-printing");
+    options.executablePath = "/tmp/chrome-linux64/chrome";
```

Comment:
I think we can remove this now that we confirmed a while ago that the Chrome Canary release didn't change behavior back then. This should also make the tests on the Windows bot work again.